### ISSUE TITLE
format the sql with inline values

### ISSUE THREAD

**Issue by favorinfo** [mtxr/vscode-sqltools#84]
### Issue Type

* [ ] Bug
* [x] Enhancement 
* [x] Feature Request
* [ ] Question
* [ ] Other

### Prerequisites (For bugfixes)

* [x] Are you running the latest version?
* [ ] Did you check the logs?
* [ ] Did you check the Setup?

### Description

I've get the sql command like `insert a(k,k,k) values (a,a,a),(b,b,b),(c,c,c);`

Once I formatted the sql commands, it ran out 
```
insert into
  a (
    k,
    k,
    k
  )
values
  (
    a,
    a,
    a
  ),
  (
    b,
    b,
    b
  );
```

is there anyways to format the commands to 

```
insert into a(k,k,k) values
(a,a,a),
(b,b,b);
```

### Steps to Reproduce (For bugfixes)

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expected to happen]

**Actual behavior:** [What actually happened]

### Versions

- *Version*:v0.13.0
- *OS*: MacOS
- *SGDB*: MySQL

You can get this information from executing `SQLTools: Version` from VSCode `Quick Open`.

**Comment by mtxr** [comment]
![](https://www.forecast.it/favicon-16x16.png) [New card](https://app.forecast.it/project/P-1/workflow/C-1)

**Comment by mtxr** [comment]
Will be fixed in v0.15.1 as seen here:
![image](https://user-images.githubusercontent.com/707561/45925023-be0a4d80-bee3-11e8-984c-2d7d352d9ea7.png)
